cli/Cargo.toml

```
@@ -14,6 +14,8 @@ config = "0.14.0"
 crossterm = "0.27"
 derive_more = "0.99.17"
 dirs = "5.0.0"
+dotenv = "0.15.0"
```

we can remove this, not referenced in the codebase anywhere
```suggestion
dotenv = "0.15.0"
```

---

Fixed